modules/ROOT/pages/manage-index.adoc

```diff
@@ -1,13 +1,34 @@
 = Manage Runtime Fabric
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
 
-After installing and configuring Runtime Fabric and deploying Mule applications and API proxies, there are additional tasks you may need to perform.
+After you install and configure Runtime Fabric and deploy Mule applications and API proxies, there are additional tasks you might need to perform.
+
+* Monitor and View
+
+** View Runtime Fabric status and resource usage. See xref:resource-usage.adoc[View Runtime Fabric Status and Resources].
+** Use Anypoint Monitoring and Anypoint Visualizer to monitor applications deployed to Runtime Fabric. See xref:manage-monitor-applications.adoc[Monitor Applications Deployed to Runtime Fabric].
+** Monitor the performance and health of your Runtime Fabric installation. See xref:using-opscenter.adoc[Use Ops Center on Anypoint Runtime Fabric].
+
+* Update
+
+** Install rtfctl to manage Runtime Fabric settings. See xref:install-rtfctl.adoc[Install rtfctl].
+** Install patches and perform other scheduled maintenance on Runtime Fabric. See xref:install-patches.adoc[Perform Scheduled Maintenance].
+** Replace your Mule license key. See xref:replace-license-key.adoc[Replace a Mule License Key].
```

Comment:
ditto

I will stop marking this; please implement GLOBALLY below and in all your documentation.